### DISCUSSION TITLE
FreeRTOS with Wio Terminal | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#647] [Q&A]
# FreeRTOS with Wio Terminal | Seeed Studio Wiki

FreeRTOS with Wio Terminal

https://wiki.seeedstudio.com/Software-FreeRTOS/

**Comment by rmrfus** [comment]
None of mentioned solutions work with Seeed rpcWiFi. This problem is years old and never being solved.
